notebooks/concepts/imviz_advanced_aper_phot.ipynb

```diff
@@ -0,0 +1,358 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2071b1be-ee74-41c3-bb36-02eb1eb4f0a8",
+   "metadata": {},
+   "source": [
+    "Even when unrotated, a different pixel scale does affect the projected mask dimension. When rotated, it is not even a rectangle anymore; furthermore, even though this projection follows WCS linking, the array from `to_mask()` does not account for rotation that we see in the viewer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b69abb83",
+   "metadata": {},
+   "source": [
+    "### What is aperture photometry telling us now?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d5ca3ec6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phot_plugin = imviz.plugins[\"Imviz Simple Aperture Photometry\"]._obj"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8fc3202b",
+   "metadata": {},
+   "source": [
+    "To re-calculate for a different subset, uncomment the desired Subset and re-run affected cells."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2eee32c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phot_plugin.subset_selected = \"Subset 1\"\n",
+    "#phot_plugin.subset_selected = \"Subset 2\"\n",
+    "#phot_plugin.subset_selected = \"Subset 3\"\n",
+    "#phot_plugin.subset_selected = \"Subset 4\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dfbaf5f1",
+   "metadata": {},
+   "source": [
+    "Calculate for the same Subset for all the data.\n",
+    "\n",
+    "Original data has a mean background of 5. Block-reduced data will have mean background of 20 (4 pixels combined into 1 pixel, while preserving total flux)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5557c11f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "phot_plugin.bg_subset_selected = 'Manual'\n",
+    "phot_plugin.background_value = 5.0\n",
+    "\n",
+    "phot_plugin.dataset_selected = \"gauss100_fits_wcs[PRIMARY,1]\"\n",
+    "phot_plugin.vue_do_aper_phot()\n",
+    "\n",
+    "phot_plugin.background_value = 20.0\n",
+    "\n",
+    "phot_plugin.dataset_selected = \"gauss100_fits_wcs_block_reduced[PRIMARY,1]\"\n",
+    "phot_plugin.vue_do_aper_phot()\n",
+    "\n",
+    "phot_plugin.dataset_selected = \"gauss100_fits_wcs_block_reduced_rotated[PRIMARY,1]\"\n",
+    "phot_plugin.vue_do_aper_phot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94ed228d",
+   "metadata": {},
+   "source": [
+    "Look at the results. The aperture sum should be close to the following numbers regardless of data if Subset is handled correctly:\n",
+    "\n",
+    "| Subset name | Aperture sum | Dev note |\n",
+    "| --- | --- | --- |\n",
+    "| Subset 1 | 738.8803424408962 | |\n",
+    "| Subset 2 | 348.33262363800094 | |\n",
+    "| Subset 3 | 857.5194857987592 | |\n",
+    "| Subset 4 | 762.3263959884644 | Why no agreement for rectangle? |"
```

Comment:
@larrybradley , I don't understand why rectangle is not behaving. Might have to pick your brain when your power comes back on...

![Screenshot 2023-08-08 175431](https://github.com/spacetelescope/jdaviz/assets/2090236/28b19184-86ce-495b-97fb-bd34d9b1cdf5)

Reply:
Figured it out. My bad https://github.com/spacetelescope/jdaviz/issues/2357